cli/src/cluster_query.rs

```diff
@@ -1492,7 +1494,7 @@ pub fn process_ping(
     let mut confirmed_count: u32 = 0;
     let mut confirmation_time: VecDeque<u64> = VecDeque::with_capacity(1024);
 
-    let mut blockhash = rpc_client.get_latest_blockhash()?;
+    let mut blockhash = tps_client.get_latest_blockhash()?;
```

Comment:
Given that it's a relatively trivial implementation, to avoid further code churn here and possibly on future changes, I'd prefer to just leave this as is.